src/components/Book.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const Book = ({ book, onDelete }) => {
```

Comment:
- [ ] you are not calling this component anywhere please make sure to implement it as intended and described in my previous comments.